src/equiv.js

```diff
@@ -13,6 +13,31 @@ export default ( function() {
 		return obj.__proto__;
 	};
 
+	function lookupDescriptor( obj, keyName ) {
+		let current = obj;
+		do {
+			const descriptor = Object.getOwnPropertyDescriptor( current, keyName );
+			if ( descriptor !== undefined ) {
+				return descriptor;
+			}
+			current = getProto( current );
+		} while ( current !== null );
+		return null;
+	}
+
+	function hasSharedDescriptor( a, b, keyName ) {
+		var aDescriptor = lookupDescriptor( a, keyName );
+		var bDescriptor = lookupDescriptor( b, keyName );
```

Comment:
This ignores where on the prototype chain the keyName property is defined when checking if it is "shared"... if that is intentional, then it should be documented and tested.